### ISSUE TITLE
missing $term

### ISSUE THREAD

**Issue by iddq** [jonathanstowe/Term-Cap#11]
https://github.com/jonathanstowe/Term-Cap/blob/774247157569fbdbac994621a5ba4e9f56a56c63/Cap.pm#L254

please use -l -1 instead of -C

-C doesn't list all capabilities like u7 of xterm

**Comment by jonathanstowe** [comment]
Yep, I think I'll have a poke round the options because it would be nice to be able to get long names *and* classic names but that may not work on all platforms.

Cheers.

/J\

**Comment by iddq** [comment]
it's horrible, il1 is still missing from the list. maybe the "-l -1" is the best output for us.

**Comment by jonathanstowe** [comment]
Sorry I totally forgot about this,  I'd probably go with `infocmp -C -r -T -l` (the formatting doesn't matter,) but it remains to be seen how well supported that is.

**Comment by marcespie** [comment]
infocmp -CTrx is the way to go.
With it, I managed to get the exact same set of capabilities from xterm (in particular, stuff like AX, Rs, ...)

See user_caps(5) for why it's needed.

**Comment by mauke** [comment]
Any news on this? I'd love to have access to `ZH`/`ZR`.
